app/src/main/java/com/moviefiend/torian/moviefiend/network/NowPlayingResponse.java

```diff
@@ -0,0 +1,34 @@
+package com.moviefiend.torian.moviefiend.network;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+
+public class NowPlayingResponse {
+    @SerializedName("results")
+    private ArrayList<MovieInfo> movies;
```

Comment:
If you're already using `@SerializedName` then you might as well use the `m` notation for the member variables.